src/daft-plan/src/physical_ops/scan.rs

```diff
@@ -0,0 +1,21 @@
+use std::sync::Arc;
+
+use daft_scan::ScanTask;
+
+use crate::PartitionSpec;
+use serde::{Deserialize, Serialize};
+
+#[derive(Clone, Debug, Serialize, Deserialize)]
+pub struct TabularScan {
+    pub scan_tasks: Vec<ScanTask>,
+    pub partition_spec: Arc<PartitionSpec>,
```

Comment:
(Offline discussion) Also, this can't be the actual spec since we don't yet have the actual values. We actually just want to capture the PartitionKeys here (`Arc<Vec<PartitionField>>`)

Reply:
Correction here. This is a **Physical Op**, and is actually indeed going to be the "Clustering Spec". This is created during logical->physical translation: https://github.com/Eventual-Inc/Daft/pull/1559/files#diff-abc819095f64baaa47a1d26c1633c44897e88ec2fb4bff32357b6aef2226aedcR91-R94